wetr.F90

```diff
@@ -38,6 +41,7 @@ subroutine getwetr(carma, igroup, rh, rdry, rwet, rhopdry, rhopwet, rc, h2o_mass
     real(kind=f), intent(in), optional   :: h2o_mass!! water vapor mass concentration (g/cm3)
     real(kind=f), intent(in), optional   :: h2o_vp  !! water eq. vaper pressure (dynes/cm2)    
     real(kind=f), intent(in), optional   :: temp    !! temperature [K]
+    real(kind=f), intent(in), optional   :: kappa   !! weight percent [%]
```

Comment:
kappa is hygroscopicity parameter, not weight percent

Reply:
Hi Mike, have you checked if this has been changed?

Reply:
This has been updated.